src/app/shared/service/survey-create.service.ts

```diff
@@ -0,0 +1,12 @@
+import { DataService } from './data.service';
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+
+@Injectable({
+  providedIn: 'root'
```

Comment:
Why did this create survey service put in shared module ? This type of services should be in core module/Admin Module